spesifikasjoner/afpant/afpant-kjoepekontrakt/kjoepekontrakt-loesningsbeskrivelse.md

```diff
@@ -9,7 +9,14 @@ Kjøpekontrakt eller informasjon fra kjøpekontrakten skal kunne oversendes fra
 
 En bank kan sende forespørsel om kjøpekontrakt til en megler basert på kjøpers fødselsnummer (11 siffer). Megler vil besvare forespørselen med en forsendelse som inneholder strukturerte data, samt en signert versjon av den fulle kontrakten. Dersom den faktiske kjøpekontrakten ikke er signert, skal kun den strukturerte delen returneres. Dersom forespørselen ikke kan besvares, vil banken få en feilmelding i retur som beskriver hvorfor megler ikke kan besvare forespørselen.
 
-# Informasjonsflyt
+Løsningen kjøpekontrakt er delt i to deler:
+|Del|Beskrivelse|
+|----|------|
+|[1](https://github.com/bitsnorge/e-tinglysing-afpant/blob/master/spesifikasjoner/afpant/afpant-kjoepekontrakt/kjoepekontrakt-loesningsbeskrivelse.md#informasjonsflyt-(del-1))|Bank forespør kjøpekontrakt og megler sender. Megler pusher deretter endringer og tillegg i kjøpekontrakten uoppfordret, til bank som allerede har forespurt kjøpekontrakten.  |
```

Comment:
samme som forrige kommentaren min, kanskje endre teksten til noe sånt som "Bank forespør kjøpekontrakt på fødselsnr og megler sender info om kjøpekontrakter tilbake"? etc

Reply:
Innarbeidet. @koeide-visma hvis det eksisterer 2 oppdrag på kjøper hos megleren, og megler da sender begge kjøpekontraktene til banken: vil megler også sende løpende endringer på begge kjøpekontraktene til banken, selv om banken bare hadde behov for det ene oppdraget? (hvis ikke jeg misforsto den forrige kommentaren)